cpe/openscap-cpe-oval.xml

```diff
@@ -699,6 +748,20 @@
                 <pattern operation="pattern match">^VERSION=.([[:digit:]]*)</pattern>
                 <instance operation="greater than or equal" datatype="int">1</instance>
             </textfilecontent54_object>
+            <file_object xmlns="http://oval.mitre.org/XMLSchema/oval-definitions-5#unix" version="1" id="oval:org.open-scap.cpe.amznlinux-release:obj:1" >
+                <filepath>/etc/os-release</filepath>
+            </file_object>
+            <textfilecontent54_object
```

Comment:
Amazon Linux is derived from RHEL and uses RPM packages. Why don't you use rpminfo_object? We do that for RHEL or CentOS. What's the reason for parsing /etc/os-release here?